tests/integration/requirements.txt

```diff
@@ -1,3 +1,4 @@
+cosl
```

Comment:
Talking with @gregory-schiano we might not need this, as PYDEPS is respected by charmcraft when building the charm and that should be baked in?

Reply:
The tests themselves need it because they import the charm, which imports the lib